src/main/pages/user-management/user-management.adoc

```diff
@@ -21,11 +21,11 @@ To find your Keycloak URL:
 
 If Che is deployed on OpenShift:
 
-* Go to OpenShift web console > keycloak namespace 
+* Go to *OpenShift web console > keycloak namespace*. 
```

Comment:
There's no such GUI element as "OpenShift web console" -- I believe this should look something like:

 * Go to the OpenShift web console and navigate to the **Keycloak** namespace.

(I checked, and there's no page/element that would be called "keycloak namespace".)